cypress/e2e/api/homepage_api_tests.cy.js

```diff
@@ -0,0 +1,34 @@
+
+describe('Successfully retrieves customers (e.g., checks for the 200 status code)', () => {
+
+    it('successfully retrieves a list of customers*', () => {
+        cy.api_retrieveCustomers()
```

Comment:
Agree it's pretty simple. The reason I had decided abstract away was because I find it to be way neater if all the api implementation live in a separate file, rather than dealing with a mix of some living inside the tests and others in a separate file. What are your thoughts on this @wlsf82?

Reply:
I will always prefer simplicity.

If the Cypress abstractions were unclear, I'd abstract it away to make it simpler.

But since this is not the case, I'd leave it in the test file.

Reply:
Thanks Walmyr, agree I have undone my abstraction.